query/src/clusters/cluster.rs

```diff
@@ -89,13 +93,38 @@ impl ClusterDiscovery {
 
     pub async fn discover(&self) -> Result<Arc<Cluster>> {
         match self.api_provider.get_nodes().await {
-            Err(cause) => Err(cause.add_message_back("(while cluster api get_nodes).")),
+            Err(cause) => {
+                label_counter_with_val_and_labels(
```

Comment:
I think it is required to add tenant id, cluster id and fligh-api-address as metrics label for both counter and gauge

Reply:
updated

Reply:
thanks :)